src/main/java/teammates/ui/controller/FileDownloadResult.java

```diff
@@ -36,12 +37,25 @@ public void send(HttpServletRequest req, HttpServletResponse resp) throws IOExce
          *     to make the servlet aware of the specified charset encoding
          */
         resp.setContentType("text/csv; charset=UTF-8");
-        resp.setHeader("Content-Disposition", "attachment; filename=\"" + fileName + ".csv\"");
+        resp.setHeader("Content-Disposition", getContentDispositionHeader());
         PrintWriter writer = resp.getWriter();
         writer.write("\uFEFF");
         writer.append(fileContent);
     }
     
+    public String getContentDispositionHeader() {
```

Comment:
Add a header comment here as well, in particular not everyone will be familiar with what Content-Disposition is.